lldb/test/API/tools/lldb-server/attach-wait/TestGdbRemoteAttachWait.py

```diff
@@ -52,6 +52,9 @@ def test_attach_with_vAttachWait(self):
         server = self.connect_to_debug_monitor()
         self.do_handshake()
 
+        if self._run_args:
+            self._run_args[0] = lldbutil.install_to_target(self, self._run_args[0])
```

Comment:
Please do this inside `_set_up_inferior`, before assigning to `_run_args`

Reply:
I have updated the patch. Thanks.